dev/doc/MERGING.md

```diff
@@ -111,7 +111,8 @@ The merge script passes option `-S` to `git merge` to ensure merge commits
 are signed. Consequently, it depends on the GnuPG command utility being
 installed and a GPG key being available. Here is a short tutorial to
 creating your own GPG key:
-<https://ekaia.org/blog/2009/05/10/creating-new-gpgkey/>
+<https://ekaia.org/blog/2009/05/10/creating-new-gpgkey/>. To link it with
```

Comment:
Why do we link some random blog instead of https://help.github.com/articles/generating-a-new-gpg-key/?

Reply:
Because that's the doc recommended by Debian. But it feels like a good idea to change it to the link you gave. Especially since it includes "step 14: add the GPG key to your GitHub account".